tests/conftest.py

```diff
@@ -36,6 +39,17 @@ def clean_logging_handlers_fixture(request):
     clean_logging_handlers()
 
 
+# This automatically used fixture changes location in which
+# ConfigManager looks up connections.toml file to non-existing location.
+# This causes it to behave as the file was not provided
+# Reason: if connections.toml is found, it automatically overrides "connections"
+# section in config.toml, which causes tests to fail.
+@pytest.fixture(autouse=True)
+def remove_connections_toml_from_config():
+    change_connections_toml_path_in_config_manager(Path("this/path/does/not/exist"))
+    yield
```

Comment:
Can't we instead of that set SNOWFLAKE_HOME to temporary/dir before tests (loading snowcli)? For test purposes we can have a fixture like `change_snowflake_home` that would allow us to switching between different config/connection configurations? WDYT?

Reply:
I was thinking about that, but unfortunately `CONFIG_MANAGER` is created with connector.constants (including snowflake home), which are determined during imports (which happens even before pytest manages to start up, at least according to multiple prints I've added here and there). We would have to set up `SNOWFLAKE_HOME` environment variable to some temporary directory before pytest command is run.
Fixtures happen too late for this to work :(
(PS I've tried mocking stuff as well, no success)